### DISCUSSION TITLE
Can we consider an alternative navigation for panning the scene?

### DISCUSSION THREAD

**Discussion by hubertp** [enso-org/enso#6089] [Work In Progress - GUI] [answered]
The current documentation lists `MMB` and two fingers for panning the scene.
This is **very** cumbersome when someone (like me) uses trackball. So I end up using zoom-in/-out for moving around since it is easier but also wrong.

Pressed key + mouse movement would be an ideal alternative. I think that was even possible in the past.

**Comment by wdanilo** [comment, marked as answer]
I believe that `ctrl + scroll` works, at least it works on my end.
EDIT: I misunderstood the question. It was about panning, not zooming. Ticket has been created, closing the discussion.

**Reply by hubertp** [reply]
@wdanilo for zooming in/out, yes. But I'm asking about the panning the scene action. `ctrl+move the mouse` doesn't do anything. You have to use `MMB` instead of `ctrl`

**Reply by wdanilo** [reply]
Oh, oh, panning! Sorry, I misread it! Yes, you are right. I like the idea of `ctrl + mouse move`, let's add it :) 
EDIT: I was looking at the code thinking this might be one-line change, but it's not. Our navigator currently does not use key bindings and uses mouse bindings in a super tacky way (it was implemented back when we did not have proper mouse bindings). So probably someone will spend on it 3-4 hours with refactoring.

**Reply by hubertp** [reply]
Ok, I will add a ticket for that.

**Reply by hubertp** [reply]
https://github.com/enso-org/enso/issues/6098
